wasm-build/wordpress-data/prepare-wordpress.sh

```diff
@@ -1,16 +1,18 @@
 #!/bin/bash
 
-# Remove previous WordPress installation
-rm -rf volume/*
+# set -e
 
-# set -e;
+cd preload
 
-cd preload;
+# Remove previous WordPress installation
+rm -rf wordpress
+rm -rf wordpress-static
 
-# Download WordPress
-wget https://wordpress.org/wordpress-6.0.1.tar.gz
-tar -xzf wordpress-6.0.1.tar.gz
-rm wordpress-6.0.1.tar.gz
+# Download latest version of WordPress
+wp_tarfile=wordpress-latest.tar.gz
```

Comment:
I feel pretty confident about defaulting to the latest minor release, e.g. `6.0.2`. Auto-upgrading to the next major release like `6.1.0` could break the build, though, if anything essential changes. I'd rather do this manually every few months until there are more checks and balances in place.

Reply:
Makes sense - that would be more stable and reliable than always pulling the newest version automatically. OK, updated.